pkgs/applications/misc/inspectrum/default.nix

```diff
@@ -1,18 +1,35 @@
-{ stdenv, fetchFromGitHub, pkgconfig, cmake, boost, fftwFloat, qt5, gnuradio }:
+{ stdenv
+, fetchFromGitHub
+, pkgconfig
+, cmake
+, boost
+, fftwFloat
+, qt5
+, gnuradio
+, liquid-dsp
+}:
 
 stdenv.mkDerivation rec {
   name = "inspectrum-${version}";
-  version = "20160403";
+  version = "20170218";
```

Comment:
Version 0.2 has some bugs, which are fixed in latest git. So the question is if it's preferred to have the version with the tag or the one containing the bugfix...